### DISCUSSION TITLE
Question about code of py_eddy_tracker_poly

### DISCUSSION THREAD

**Discussion by NTUBobWu** [AntSimi/py-eddy-tracker#151] [Q&A]
Hello
I found that when I use the code of plot, I cannot execute it. The output shows that "No module named 'Polygon' ", so I think there is a module called 'Polygon'. However, I did not find the module, so I want to ask you how I can solve this problem or how I can do to execute the code of poly.
Thanks.

**Comment by AntSimi** [comment]
Hello,
I don't know which code did you run, and i don't know how you install pyeddytracker, and on which platform?

**Reply by NTUBobWu** [reply]
Hi,
I download 'Master' ZIP and use spyder to run the pet_eddy_detection.py. This is the result I get.
![image](https://user-images.githubusercontent.com/108706842/177277229-446cb877-e61f-43d8-92d9-633d33edd898.png)
It shows that there is not a module called Polygon in \Users\User\Desktop\eddy\py_eddy_tracker\poly.py.

**Reply by AntSimi** [reply]
did you follow this step https://py-eddy-tracker.readthedocs.io/en/stable/installation.html?

**Reply by NTUBobWu** [reply]
I did not. I do not know how to use pip on spyder. I just download your master ZIP and put your code on spyder.
Can you teach me how to set it up?
I do not know how pip works.

**Reply by NTUBobWu** [reply]
I think I can run your code correctly if the code doesn't contain the function ''poly''.

**Reply by AntSimi** [reply]
Sorry but i am not a spyder user
